updates to scheduler, jobs, wf, reminders docs (#4514)
dapr/docs: daprdocs/content/en/developing-applications/building-blocks/actors/actors-timers-reminders.md

## Actor reminders

{{% alert title="Note" color="primary" %}}

> In Dapr v1.15, actor reminders are stored by default in the [Scheduler service]({{< ref "scheduler.md#actor-reminders" >}}). The actual API surface that you use to author Reminders/Timers for Actors hasn't changed and will continue to be available. All existing reminders are automatically migrated to the Scheduler service with _no_ loss of reminders as a one time operation for each actor type.

Reviewer: ```suggestion
In Dapr v1.15, actor reminders are stored by default in the [Scheduler service]({{< ref "scheduler.md#actor-reminders" >}}). When upgrading to Dapr v1.15 all existing reminders are automatically migrated to the Scheduler service with no loss of reminders as a one time operation for each actor type.
```
